### DISCUSSION TITLE
Crashed during the process due to Out of memory

### DISCUSSION THREAD

**Discussion by manuelpc2002** [AloneLiberty/FlipperNested#7] [Q&A] [answered]
During the process in the PC FlipperNested was stopped because of a crash:
C:\Users\Papa>FlipperNested
[?] Checking 940566C1.nonces
Recovering key type A, sector 14
[=] Hardnested attack starting...
[=] ---------+---------+---------------------------------------------------------+-----------------+-------
[=]          |         |                                                         | Expected to brute force
[=]  Time    | #nonces | Activity                                                | #states         | time
[=] ---------+---------+---------------------------------------------------------+-----------------+-------
[=]        0 |       0 | Start using 4 threads                                   |                 |
[=]        0 |       0 | Brute force benchmark: 21 million (2^24,3) keys/s       | 140737488355328 |   77d
[=]        1 |       0 | Using 235 precalculated bitflip state tables            | 140737488355328 |   77d
[!!] Out of memory error in init_nonce_memory(). Aborting...

[!!!] Something went VERY wrong in key recovery.
You MUST report this to developer!
[-] No keys found!

**Comment by AloneLiberty** [comment]
Which CPU and how much RAM you have?

**Reply by manuelpc2002** [reply]
![image](https://user-images.githubusercontent.com/4020854/236690839-533db271-9262-4457-804f-eb6c48a9a01a.png)
Intel(R) Core(TM) i7-3555LE CPU @ 2.50GHz   2.49 GHz
RAM: 4GB

**Reply by AloneLiberty** [reply, marked as answer]
Try closing all other applications or using Linux livecd without environment
